jsonrpc-specs/src/methods/discv5.json

```diff
@@ -0,0 +1,567 @@
+			"required": true,
+			"schema": {
+				"title": "nodeInfoResults",
+				"description": "ENR and NodeId of the local peer",
+				"type": "object",
+				"required": [
+					"enr",
+					"nodeId"
+				],
+				"properties": {
+					"enr": {
+						"title": "nodeENR",
+						"description": "URL-safe base64 encoded \\\"text\\\" version of the ENR prefixed by \\\"enr:\\\".\"",
+						"$ref": "#/components/schemas/Enr"
+					},
+					"nodeId": {
+						"title": "nodeId",
+						"description": "Hex encoded `NodeId` of an ENR (a 32 byte identifier).",
+						"$ref": "#/components/schemas/bytes32"
+					}
+				}
+			}
+		}
+	},
+	{
+		"name": "discv5_routingTableInfo",
+		"summary": "Returns meta information about discv5 routing table.",
+		"params": [],
+		"result": {
+			"name": "routingTableInfoResult",
+			"description": "Discv5 routing table information",
+			"required": true,
+			"schema": {
+				"title": "routingTableInfoResults",
+				"description": "Routing table details",
+				"type": "object",
+				"required": [
+					"localKey",
+					"buckets"
+				],
+				"properties": {
+					"localKey": {
+						"title": "nodeId",
+						"description": "The key identifying the local peer that owns the routing table.",
+						"$ref": "#/components/schemas/bytes32"
+					},
+					"buckets":  {
+						"title": "kBucketsTable",
+						"description": "Represents a Kademlia routing table.",
+						"$ref": "#/components/schemas/kBuckets"
+					}
+				}
+			}
+		}
+	},
+	{
+		"name": "discv5_setEnr",
+		"summary": "Write an ethereum node record to the database.",
+		"params": [
+			{
+				"name": "enr",
+				"required": true,
+				"schema": {
+					"title": "Ethereum node record",
+					"$ref": "#/components/schemas/Enr"
+				}
+			}
+		],
+		"result": {
+			"name": "setEnrResult",
+			"description": "Returns ethereum node record if the node record has been successfully saved.",
+			"schema": {
+				"title": "Ethereum node record",
+				"$ref": "#/components/schemas/Enr"
+			}
+		}
+	},
+	{
+		"name": "discv5_getEnr",
+		"summary": "Fetch the latest ENR associated with the given node ID",
+		"params": [
+			{
+				"name": "node_id",
+				"required": true,
+				"schema": {
+					"title": "NodeId",
+					"$ref": "#/components/schemas/bytes32"
+				}
+			}
+		],
+		"result": {
+			"name": "getEnrResult",
+			"description": "Returns latest ENR associated with the given node ID.",
+			"schema": {
+					"title": "Ethereum node record",
+					"$ref": "#/components/schemas/Enr"
+				      }
+		}
+	},
+	{
+		"name": "discv5_deleteEnr",
+		"summary": "Delete a Node ID from the local database",
+		"params": [
+			{
+				"name": "node_id",
+				"required": true,
+				"schema": {
+					"title": "NodeId",
+					"$ref": "#/components/schemas/bytes32"
+				}
+			}
+		],
+		"result": {
+			"name": "deleteEnrResult",
+			"description": "Returns None upon successful deletion of the node record.",
+			"schema": {
+				"type": "string",
+				"pattern": "None$"
```

Comment:
What is a `None` here?

Reply:
It is a regex expression matching the `None` string but probably it makes more sense to return a boolean instead?